example/res/layout/payment_activity.xml

```diff
@@ -1,27 +1,90 @@
 <?xml version="1.0" encoding="utf-8"?>
 <LinearLayout
     xmlns:android="http://schemas.android.com/apk/res/android"
-    android:layout_width="fill_parent"
-    android:layout_height="fill_parent"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent"
+    android:paddingLeft="8dp"
+    android:paddingStart="8dp"
+    android:paddingRight="8dp"
+    android:paddingEnd="8dp"
+    android:gravity="center_horizontal"
     android:orientation="vertical">
 
     <TextView android:id="@+id/payment_form_title"
               android:text="@string/addPayment"
               style="@style/Header" />
 
-    <fragment android:id="@+id/payment_form"
-              android:layout_width="fill_parent"
-              android:layout_height="wrap_content"
-              class="com.stripe.example.fragment.PaymentFormFragment"/>
+    <EditText
+        android:id="@+id/number"
+        android:layout_width="match_parent"
+        android:layout_height="wrap_content"
+        style="@style/TextField"
```

Comment:
Also, it's much more common to use styles for groups of things, as opposed to a style that includes text specific to a single control.